### DISCUSSION TITLE
Quizlet clone

### DISCUSSION THREAD

**Discussion by Adam81-creator** [t3-oss/create-t3-turbo#1261] [Q&A]
Hello, this is going to sound like a dumb question but can someone tell me how to upload this Quizlet clone to my domain? 
I can't find any walkthroughs anywhere.
Thank you

**Comment by noxify** [comment]
If you plan to use vercel to host your app, it should be enough to follow the [Deployment Guideline](https://github.com/t3-oss/create-t3-turbo?tab=readme-ov-file#deployment)
